main.go

```diff
@@ -118,21 +127,46 @@ func main() {
 
 	stopCh := make(chan struct{})
 
-	// setup key cert reloader
-	certReloader, err := r.NewCertReloader(r.ReloadConfig{
-		KeyFile:  *key,
-		CertFile: *cert,
-	}, stopCh)
+	nTokenPeriod, err := time.ParseDuration(*nTokenExpireTime)
 	if err != nil {
-		log.Panicf("Error occurred when creating new reloader. Error: %v", err)
+		log.Panicf("NToken expiry duration input is invalid. Error: %v", err)
 	}
 
-	// zmsClient setup for API call
-	zmsClient, err := createZMSClient(certReloader, *zmsURL, *disableKeepAlives)
-	if err != nil {
-		log.Panicf("Error occurred when creating zms client. Error: %v", err)
+	var zmsClient *zms.ZMSClient
+	if *useNToken {
+		client := zms.NewClient(*zmsURL, nil)
```

Comment:
could we remove `client` temporary var?

Reply:
If I remove this line and replace directly with zmsClient = &zms.NewClient(*zmsURL, nil), I get an error of "cannot take the address of zms.NewClient(*zmsURL, nil)" for some reason.

Reply:
Ah yep, go doesn't allow taking the address of a function return directly unfortunately so we need the temporary variable.